### PR TITLE
53 design the user interface for input

### PR DESCRIPTION
**Pull request by ndefokou**
@stephane-segning

### CODE REVIEW

File: templates/input.html
Comment (stephane-segning) [inline templates/input.html:1]: Cancel changes that were made here.

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good job. Just this small part remains

**Comment by stephane-segning** [comment]
Please link a ticket

**Comment by nancymuyeh** [comment]
@stephane-segning merge conflict resolved

**Review by stephane-segning — APPROVED** [review]
Good work @ndefokou @nancymuyeh !

**Pull request merged**